Quarkus KafkaStream application do not close immediately when no broker is present

**Describe the bug**
Quarkus application using the kafka-stream extension do not close immediately if the kafka broker is not available at the time the application starts. 

**Expected behavior**
If no broker available the application have to shutdown immediately

**Actual behavior**
When closing the application, the admin client that is used by the kafkastream extension do not close immediately because it waits to perform startup task (querying metadata) that need to timeout before closing the admin client.


**To Reproduce**
* Start a kafkastream application without starting any kafka broker 
* Quit the application (with Ctrl-C)
* Wait (60s) and read the following message : 

`
2020-11-16 12:05:28,601 WARN  [org.apa.kaf.cli.NetworkClient] (kafka-admin-client-thread | adminclient-1) [AdminClient clientId=adminclient-1] Connection to node -1 (localhost/127.0.0.1:9092) could not be established. Broker may not be available.
2020-11-16 12:05:29,410 INFO  [org.apa.kaf.cli.adm.int.AdminMetadataManager] (kafka-admin-client-thread | adminclient-1) [AdminClient clientId=adminclient-1] Metadata update failed: org.apache.kafka.common.errors.TimeoutException: Call(callName=fetchMetadata, deadlineMs=1605524729409) timed out at 1605524729410 after 1 attempt(s)
Caused by: org.apache.kafka.common.errors.TimeoutException: Timed out waiting for a node assignment.
2020-11-16 12:05:29,773 INFO  [org.apa.kaf.cli.adm.int.AdminMetadataManager] (kafka-admin-client-thread | adminclient-1) [AdminClient clientId=adminclient-1] Metadata update failed: org.apache.kafka.common.errors.TimeoutException: Call(callName=fetchMetadata, deadlineMs=1605524759410) timed out at 9223372036854775807 after 1 attempt(s)
Caused by: org.apache.kafka.common.errors.TimeoutException: The AdminClient thread has exited.
`
Regards

--- a/extensions/kafka-streams/runtime/src/main/java/io/quarkus/kafka/streams/runtime/KafkaStreamsProducer.java
+++ b/extensions/kafka-streams/runtime/src/main/java/io/quarkus/kafka/streams/runtime/KafkaStreamsProducer.java
@@ -122,7 +122,7 @@ public class KafkaStreamsProducer {
             kafkaStreams.close();
         }
         if (kafkaAdminClient != null) {
-            kafkaAdminClient.close();
+            kafkaAdminClient.close(Duration.ZERO);
         }
     }
 

